### ISSUE TITLE
Request - Addon for Web browser

### ISSUE THREAD

**Issue by usercase** [spmbt/googleSearchExtraButtons#21]
Hi,

Thank you for your work. 

As I understand, there is a problem to save user's script settings. 

Could it be easier to develop an Addon to solve this problem ? 

Nowadays, the addon from a browser can be used on an other, you could develop once for all browser.

Cheers

**Comment by spmbt** [comment]
Yes, addon will save settings in backgound.js and will not need third domain name for it.
But I have not automatic deploy of script to userscript and extensions for different browsers, so I strive to stay within one instance of crossbrowser code.
